jerry-core/parser/js/js-parser-tagged-template-literal.h

```diff
@@ -16,7 +16,7 @@
 #ifndef ECMA_TAGGED_TEMPLATE_LITERAL_H
 #define ECMA_TAGGED_TEMPLATE_LITERAL_H
 
-/* \addtogroup parser Parser
+/** \addtogroup parser Parser
```

Comment:
My gut feeling is that the group-closing braces are at the wrong place at the end of this file. (They should come before the `#endif`.)